qc_grader/challenges/qdc_2025/qdc25_lab2.py

```diff
@@ -0,0 +1,169 @@
+from typeguard import typechecked
+
+from qiskit import QuantumCircuit
+from lattice import HeavyHexLattice
```

Comment:
This import needs be changed to `from .lattice import HeavyHexLattice` so that it will use the local `lattice.py` file